src/devices/Units/Pressure.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Units
+{
+    /// <summary>
+    /// Class representing pressure
+    /// </summary>
+    public static class Pressure
+    {
+        /// <summary>
+        /// The mean sea-level pressure (MSLP) is the average atmospheric pressure at mean sea level.
+        /// </summary>
+        public const double MeanSeaLevelPressure = 1013.25;
```

Comment:
I'd recommend to remove `Pressure` suffix from the name since it's already in the name of the class

Reply:
Complete rewrite made.